Library/Homebrew/brew.sh

```diff
@@ -233,6 +227,13 @@ HOMEBREW_CACHE="${HOMEBREW_CACHE:-${HOMEBREW_DEFAULT_CACHE}}"
 HOMEBREW_LOGS="${HOMEBREW_LOGS:-${HOMEBREW_DEFAULT_LOGS}}"
 HOMEBREW_TEMP="${HOMEBREW_TEMP:-${HOMEBREW_DEFAULT_TEMP}}"
 
+case "$*" in
+  --prefix)            echo "$HOMEBREW_PREFIX"; exit 0 ;;
+  --cellar)            echo "$HOMEBREW_CELLAR"; exit 0 ;;
+  --repository|--repo) echo "$HOMEBREW_REPOSITORY"; exit 0 ;;
+  --cache)             echo "$HOMEBREW_CACHE"; exit 0 ;;
```

Comment:
As mentioned: maybe just `--cache` could be here?

Reply:
Done, `--cache` is just sitting in its own little `case` block now. Hope that's fine.